spec/ask_export/survey_response_fetcher_spec.rb

```diff
@@ -0,0 +1,101 @@
+RSpec.describe AskExport::SurveyResponseFetcher do
+  describe ".call" do
+    before do
+      allow_any_instance_of(described_class).to receive(:sleep)
+      allow_any_instance_of(Faraday::Request::Retry).to receive(:sleep)
+    end
+
+    around do |example|
+      ClimateControl.modify(
+        SMART_SURVEY_API_TOKEN: "token",
+        SMART_SURVEY_API_TOKEN_SECRET: "secret",
+      ) { example.run }
+    end
+
+    let(:since_time) { Date.yesterday.noon }
+    let(:until_time) { Date.current.noon }
+
+    it "requests responses in smart survey covering the dates specified" do
+      request = stub_smart_survey_api(since_time: since_time,
+                                      until_time: until_time)
+      described_class.call(since_time, until_time)
+      expect(request).to have_been_made
+    end
+
+    it "returns a hash of a presented response from Smart Survey" do
+      response = presented_survey_response
```

Comment:
```suggestion
      presented_response = presented_survey_response
```

Minor: I had to remind myself it's not the actual response.